mailpoet/assets/js/src/settings/store/actions/mss_and_premium.ts

```diff
@@ -45,7 +45,9 @@ export function* verifyMssKey(key: string) {
   } else {
     yield setSettings(call.res.data);
     fields.mssStatus = getMssStatus(
-      call.res.data.mta.mailpoet_api_key_state.code === 200,
+      ['200', 200].includes(
+        call.res.data.mta.mailpoet_api_key_state.code as number | string,
+      ),
```

Comment:
Just curious. Could we use `parseInt(call.res.data.mta.mailpoet_api_key_state.code, 10) === 200` or maybe `Number(call.res.data.mta.mailpoet_api_key_state.code)` here?
That way, I believe, we could avoid using typecasting and Typescript should pick a correct type.